Src/Generic/Styling/TextStyle.swift

```diff
@@ -47,7 +47,7 @@ extension TextStyle {
         style.lineSpacing = 6
     }
 
-    static let bodyButtonText = TextStyle.default.restyled { (style: inout TextStyle) in
+    static let bodyText = TextStyle.default.restyled { (style: inout TextStyle) in
```

Comment:
I think remove this style and use `bodySmallSmall` instead.

Reply:
Went back as it was and put back bodyButtonText so i didn't have to change something in offerscreen change startdate button. So i just updated the payment style to bodySmallSmall :)